packages/run/README.md

```diff
@@ -72,6 +72,19 @@ export default {
 };
 ```
 
+### `allowRestarts`
+
+Type: `Boolean`<br>
+Default: `false`
+
+If `true`, instructs the plugin to listen to `stdin` for the sequences listed below followed by enter (carriage return).
+
+`restart` - Kills the currently running bundle and starts it again. _Note: This does not create a new bundle, the bundle is run again "as-is". This can be used to test configuration changes or other changes that are made without modifying your source_
```

Comment:
Im not sure what this and the `clear` bit below are describing. The type of this option is stated to be a `Boolean` so where does this come into play?

Reply:
OK, thanks. It'll be worth delineating that a bit more:

```suggestion
#### `stdin` Input Actions

When this option is enabled, `stdin` will listen for the following input and perform the associated action:

- `restart` → Kills the currently running bundle and starts it again. _Note: This does not create a new bundle, the bundle is run again "as-is". This can be used to test configuration changes or other changes that are made without modifying your source_
```

(And please copy the `- key →` format to the `clear` bit below

Reply:
In hindsight, I wrote that with my own assumptions of how this functionality works in mind without considering how someone unfamiliar with it might interpret it. I've made the changes you suggested.